qbft/spectest/tests/create_message_spectest.go

```diff
@@ -66,6 +67,8 @@ func (test *CreateMsgSpecTest) Run(t *testing.T) {
 		require.Fail(t, "post state not equal", diff)
 	}
 	require.EqualValues(t, test.ExpectedRoot, hex.EncodeToString(r[:]))
+
+	typescomparable.CompareWithJson(t, test, test.TestName(), reflect.TypeOf(test).String())
```

Comment:
what is the point of this if the root is being compared just above? just so we can see the diffs clearly?

Reply:
I think the above root test is just a check on the expected root of the created message.
But, the `CompareWithJson` should show us any difference in the test "content".
@GalRogozinski correct me if I'm wrong